### ISSUE TITLE
WPP.chat.forwardMessage doesnt work

### ISSUE THREAD

**Issue by Birol2010** [wppconnect-team/wa-js#1531]
## Description

Hi.
When I run WPP.chat.forwardMessage(senderId,forwardedMessagejId)
It gives an error called 
Uncaught (in promise) TypeError: o.ChatStore.forwardMessagesToChats is not a function
    at t.forwardMessage (<anonymous>:52:140610)
return await o.ChatStore.forwardMessagesToChats gives error.

## Environment

- **WhatsApp version:** X.YYYY.ZZ
- **WA-JS version(s):** 2.28
- **Browser:** EDGE last
- **OS:** Windows 10
- **MultiDevice (BETA):** yes/no

## Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

## Log Output

```
If relevant, paste all of your Log Output
```

## Your Code

```
If relevant, paste all of your challenge code in here
```

## Additional context / Screenshot

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by keyurboss** [comment]
fixed #1535

**Comment by keyurboss** [comment]
You can Test it using https://github.com/keyurboss/wa-js/tree/main

**Comment by edgardmessias** [comment]
Closed by #1535
